### PR TITLE
Displays Section 1 Impact Level On Results Page

### LINKED ISSUE #449: Does not display "Section 1 Impact Level" On Results Page
**Describe the bug - Décrivez le bogue**
Section 1 : Impact Level not displaying on the results page.

**To Reproduce - Pour reproduire**
Steps to reproduce the behavior - Étape pour reproduire le comportement:
1. Go to results page.
2. Add Section 1 to be displayed for the clients using it.
3. Remove textbox containing "Impact Level Results, Current Score, Mitigation Score, Risk Impact Score" 
4. Moving independent scores beneath this section 1 Impact Level.

### PR DESCRIPTION
**Pull request by SalimErradi**
Fixes #449

### CODE REVIEW

**Review by MrDeshaies — APPROVED** [review]
looks good. french and english. merging!

File: src/views/Results.vue [hunk 1]
```diff
@@ -50,10 +50,14 @@
     </ul>
 
     <div id="results-printable">
-      <Score />
-      <Obligations />
-
       <div class="container-fluid">
+        <h2>{{ "Section 1: " + $t("riskLevel") }} {{ ": " + score[3] }}</h2>
```
Comment (MrDeshaies) [inline src/views/Results.vue:54]: From the table of contents, the #score anchor is broken. it should point to this h2. Same for French.

**Review by MrDeshaies — CHANGES_REQUESTED** [review]
fix "score" anchor from table of contents, to jump to the h2 for score section

File: src/views/Results.vue (hunk 1 above)
Comment (SalimErradi) [reply src/views/Results.vue]: Fixed.

**Review by MrDeshaies — APPROVED** [review]

**Comment by MrDeshaies** [comment]
tbh, it had been fixed by a previous commit ;)

**Pull request merged**